src/prompt.rs

```diff
@@ -3,10 +3,10 @@ use tera::{Context, Error};
 use std::collections::HashMap;
 use tera::Tera;
 
-pub fn format_prompt(prompt: &str, map: HashMap<&str, &str>) -> Result<String, Error> {
+pub fn format_prompt(prompt: &str, lang: &str, map: HashMap<&str, &str>) -> Result<String, Error> {
     let context = Context::from_serialize(map)?;
-
-    Tera::one_off(prompt, &context, false)
+    let new_prompt = format!("REMEMBER, ANSWER IN {}.\n{}", lang.to_uppercase(), prompt);
```

Comment:
Add this to the prompts instead. This will be easier to extend in the longer term